src/v2/utils/files/FileService.ts

```diff
@@ -19,6 +19,26 @@ export class FileService {
     return resolve(process.env.BASE_URL, join(directory, fileName + extname(file.originalname)));
   }
 
+  getFileName (avatar: string) {
+    const parts = avatar.split('/');
+    return parts[parts.length - 1];
+  }
+
+  async checkAvatarExist (avatar: string) {
```

Comment:
Abstract this method, its task will simply check existence of files depending on whether they are private or static